subgraphs/uniswap-forks/package.json

```diff
@@ -5,21 +5,58 @@
     "codegen": "graph codegen",
     "build": "graph build",
 
-    "prepare:apeswap-bsc": "mustache config/apeswap/bsc.json templates/apeswap.template.yaml > subgraph.yaml",
-    "prepare:apeswap-matic": "mustache config/apeswap/polygon.json templates/apeswap.template.yaml > subgraph.yaml",
-    "prepare:uniswap-v2-mainnet": "mustache config/uniswap-v2/mainnet.json templates/uniswap.v2.template.yaml > subgraph.yaml",
+    "prepare:apeswap-bsc": "mustache config/apeswap/bsc.json templates/apeswap/apeswap.template.yaml > subgraph.yaml",
```

Comment:
This will get pretty bloated quickly. Especially if we add more forks. I was wondering if we could have something like a `deployment.json` which contains:
```
{
  "uniswap-v2": ["mainnet"],
  "sushiwap": ["mainnet", "celo", "bsc", ...],
  "spookyswap": ["fantom"],
  ...
}
```

We will have a deployment script load the content of the `deployment.json` and deploy according to the arguments.

For example, something like:
- `npm run deploy uniswap-v2 mainnet` will deploy uniswap-v2 to mainnet.
- `npm run deploy uniswap-v2` will deploy uniswap-v2 to all networks.
- `npm run deploy` will deploy all forks.

Reply:
Added this capability to the uniswap forks. Check the deployment folder and package.json.